tests/kphp_ci_pipeline_runner.py

```diff
@@ -170,10 +170,18 @@ def parse_args():
         type=str,
         dest="cxx_name",
         default="g++",
-        choices=["g++", "clang++", "clang++-16"],
+        choices=["g++", "clang++", "clang++-16", "clang++-18", "clang++-19"],
```

Comment:
I think we should throw away clang16.
The current solution raises a question: why clang16 is here, but clang17 is not?
The last release of clang16 was more than a year ago, it's obsolete now, v18 and v19 are good I think

Reply:
`clang16` is used in current `KPHP` pipelines. Removing it also requires modifying the pipelines. It's removal definitely should be a distinct step.